### DISCUSSION TITLE
Moes Tuya MS-105B zigbee

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#22148] [Ideas]
I bought and installed this device: https://nl.aliexpress.com/item/1005004986958450.html

Can it be supported by Tasmota Zigbee?

It's allready added to zigbee2mqtt: https://github.com/Koenkk/zigbee2mqtt/issues/13747

https://zigbee.blakadder.com/Moes_MS-105B.html

I'm doubting if I should replace it because it seems to use unknown zigbee commands and I want it to be compatible with philips hue or ikea when I would sell the house so the next owner can easily integrate the dimmers in is own smarthome bridge/app.

**Comment by s-hadinger** [comment]
Moes/Tuya is known to use non-standard Zigbee commands and codes. However this one seemed to be confirmed working with Tasmota as indicated in the link you provided.

**Reply by jeroenst** [reply]
Yes, it says it does but it doesn't unfortunately. I guess this one is for the trash-bin in my case.

**Reply by s-hadinger** [reply]
Will you share logs?

**Reply by jeroenst** [reply]
In the future, I removed the device at the moment.
